src/API/HealthController.ts

```diff
@@ -39,13 +66,49 @@ export class HealthController {
     }
   }
 
+  private async getBlockchainInfo(): Promise<any> {
+    try {
+      const { blockchainInfo } = (await this.collection.findOne({ name: 'blockchainInfo' })) || {
```

Comment:
I think we can just default it to an empty object, don't think we need a message.

```ts
const { blockchainInfo = {} } = await this.collection.findOne({ name: 'blockchainInfo' })
```

Reply:
If you agree, apply to the other 2 functions as well